main/models/space.py

```diff
@@ -96,3 +98,23 @@ def as_geojson_feature(self):
                 "logo": self.logo_image_url
             }
         }
+
+    # get membership type, returns: None, Supporter, Representative
```

Comment:
Expected types are `None`, `Supporter` or `Representative`. Returned values are `Member` or `None`.

Reply:
I note your `TODO:` but just want to confirm that what ever calls this won't get pissy if the value `Member` is returned when not expected.

Reply:
Oops, spaces can't be supporters or representatives, just members or not members, I'll remove those (perils of stealing code from user.py)